src/views/InvolvementProfile/components/Membership/components/MemberList/components/MemberListItem/index.js

```diff
@@ -277,24 +311,37 @@ const MemberListItem = ({
       content = (
         <Accordion defaultExpanded={(isAdmin || isSuperAdmin) && !isMobileView}>
           <AccordionSummary expandIcon={<ExpandMoreIcon />}>
-            <Grid container>
-              <Grid item xs={6} sm={7} md={8}>
-                <Typography>
-                  {member.FirstName} {member.LastName}
-                </Typography>
+            <Grid container alignItems="center" spacing={2}>
+              <Grid item xs={9} sm={10}>
+                <Grid container spacing={3} wrap="nowrap" alignItems="center">
+                  <Grid item>
+                    <IMG
+                      className="people-search-avatar-mobile"
+                      src={`data:image/jpg;base64,${avatar}`}
+                      alt=""
+                      noLazyLoad="true"
+                      placeholderColor="#eeeeee"
+                    />
```

Comment:
The `IMG` component from react-graceful-image is useful when you're loading an image from a url or filepath. However, since we're requesting the image data and storing is as a state variable, it doesn't make much sense to use `react-graceful-image` because it isn't doing what it's designed to do. You might as well use the Material-UI `Avatar` component instead (see my changes to ContactListItem for an example).